lib/pem/manager.rb

```diff
@@ -68,7 +68,9 @@ def create_certificate
         end
 
         if PEM.config[:generate_p12]
-          p12_cert_path = File.join(PEM.config[:output_path], "#{filename_base}.p12")
+          output_path = PEM.config[:output_path]
+          FileUtils.mkdir_p(File.expand_path(output_path)) unless Dir.exist?(File.expand_path(output_path))
```

Comment:
You can remote the `Dir.exist?(File.expand_path(output_path))` as `mkdir_p` will not fail if the directory is already there